fg-stitch-lib/src/align/aligners/mod.rs

```diff
@@ -46,7 +46,7 @@ use noodles::{
         alignment::Record as SamRecord,
         record::{
             cigar::op::{Kind, Op},
-            data::field::tag::{ALIGNMENT_HIT_COUNT, ALIGNMENT_SCORE, HIT_INDEX, TOTAL_HIT_COUNT},
```

Comment:
I find the NH/IH/HI tags confusing when present.  In particular that (1) the NH tag can be > than the # of alignments in the file, (2) that HI is the index of the alignment in the file (should change if we reorder the file).  So removing them.

Reply:
I think `NH` is for the scenario where the aligner is configured to only emit a subset of alignments. But I agree they are not as clear/useful as the tags you've added. I would only add them if there were a downstream tool that expected them, but we can wait for someone to open an issue.